zha/zigbee/device.py

```diff
@@ -243,6 +243,10 @@ def __init__(
             if ep_id != 0:
                 self._endpoints[ep_id] = Endpoint.new(endpoint, self)
 
+    def __repr__(self) -> str:
+        """Return a string representation of the device."""
+        return f"{self._zigpy_device.__repr__()} - quirk_applied: {self.quirk_applied} - quirk_or_device_class: {self.quirk_class} - quirk_id: {self.quirk_id}"
```

Comment:
Also, would it make more sense to use  `repr(device)` instead of `device.__repr__()` here?